lib/init.dart

```diff
@@ -19,6 +20,7 @@ Future initPolymer() async {
   await initWebComponents(
       typeFilter: [CustomElement, CustomElementProxy, PolymerRegister],
       initAll: true);
+  document.body.attributes.remove('unresolved');
```

Comment:
add a TODO since this is a temporary workaround? I assume we don't want to do this when polymer is doing it internally later on, right?